playbooks/roles/start/tasks/wait_for_zowe.yml

```diff
@@ -99,6 +109,33 @@
   until: _result.status == 204
   retries: "{{ wait_for_zowe_service_retries | int }}"
   delay: "{{ wait_for_zowe_service_delay | int }}"
+  # if zowe_zos_host and zowe_external_domain_name are not same, zlux could be
+  # started off z/OS and then we couldn't test right now
+  when: zowe_zos_host == zowe_external_domain_name and "api-mediation" in zowe_launch_components
```

Comment:
Please change to `when: zowe_zos_host == zowe_external_domain_name and ("api-catalog" in zowe_launch_components or zowe_launch_components == "")`